lib/foreman_tasks/engine.rb

```diff
@@ -186,6 +185,16 @@ class Engine < ::Rails::Engine
       ::Dynflow::ActiveJob::QueueAdapters::JobWrapper.include Actions::TaskSynchronization
     end
 
+    initializer "foreman_tasks.observable_events" do
```

Comment:
Can be dropped from this PR.

That's my attempt to fix at least some `autoloadable constant loading during initialization` warnings. Seems I was able to fix some of them in the plain Foreman, but with each plugin the number of warnings rises exponentially. Not required for Zeitwerk, but in the future Rails will make these warnings be errors.

Reply:
Is that something we can already merge today or wouldn't it make sense without Zeitwerk?

Reply:
That's experimental and I'll drop this from the PR.

I think we could deal with the warnings without Zeitwerk enabled, but the warnings appear only if we enable it. I'd postpone that after we enable it for all the plugins, maybe devs will get more interested in resolving that with us.